src/sentry/options/defaults.py

```diff
@@ -552,6 +552,7 @@
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
 register("github-login.organization", flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE)
+register("github-extension", default=False, flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
```suggestion
register("github-extension.enabled", default=False, flags=FLAG_MODIFIABLE_BOOL | FLAG_AUTOMATOR_MODIFIABLE)
```

Reply:
did you not want it to specify enabled? also, did you not want this option configurable per org?

Reply:
this is not per org, github will not send us any information besides user information

Reply:
got it

Reply:
this is mainly for blocking the api from being hit rn in prod